resources/assets/coffee/react/profile-page/header-extra.coffee

```diff
@@ -142,15 +142,18 @@ class ProfilePage.HeaderExtra extends React.Component
 
         div className: "#{bn}__column #{bn}__column--text #{bn}__column--shrink",
           div className: "#{bn}__rows",
-            @fancyLink
```

Comment:
the text was being set by `fancyLink`
![image](https://user-images.githubusercontent.com/1694544/44759350-a0f88e00-ab74-11e8-8a27-eead1e88569e.png)

I'd update `fancyLink` to set a different css class if it's not a link

Reply:
would using `:not` selector be fine for that?

Reply:
except the twitter link is in a `span` so it doesn't apply the correct colour.

Also missing the colour for the forum posts link